task-5/task-5/src/timer.ts

```diff
@@ -0,0 +1,165 @@
+export class Timer {
+  timer
+  isRunning: boolean
+  type
+  time: number
+  timeElement
+  startBtn
+  stopBtn
+  resetBtn
+  typesButton
+  focusBtn
+  shortBtn
+  longBtn
+
+  constructor(time: number) {
+    this.timer = document.querySelector<HTMLDivElement>('#app')!
```

Comment:
This needs to come from a constructor, what if there is no element with ID of app?

Reply:
I used id="app" because Vite created it by default. But I can change that to property id, so we can create more timers.